package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "freedom",
   "description": "Embracing a distributed web",
-  "version": "0.6.25",
+  "version": "0.6.26",
```

Comment:
If you want we can merge this main pull request first, cut a release, and then keep the higher dependencies on those as you have them. Else yes, ^ should suffice.